src/ai/backend/storage/ddn/__init__.py

```diff
@@ -159,7 +156,8 @@ async def create_quota_scope(
         except CalledProcessError as e:
             raise RuntimeError(f"'lfs project -p {project_id}' command failed: {e.stderr}")
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: lines 144 to 157]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->Could you tell me more about changing logic here in a way of calling `lfs`?<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/lablup/backend.ai/2145?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->

Reply:
`lfs project -p ...` command maps DDN Project ID to a directory. This enables us to set a quota to a directory, so it should be done in `create_quota_scope()` even there is no specified quota option.